src/hb-subset.cc

```diff
@@ -227,32 +234,30 @@ _add_head_and_set_loca_version (hb_face_t *source, bool use_short_loca, hb_face_
 
     hb_blob_t *head_prime_blob = hb_blob_create ((const char*) head_prime,
                                                  OT::head::static_size,
-                                                 HB_MEMORY_MODE_WRITABLE,
+                                                 HB_MEMORY_MODE_READONLY,
```

Comment:
Why?!

Reply:
head_prime_blob isn't modified after this point (just passed to add_table) so no need for it to be writable.